mlos_bench/db/db_schema_sqlite.sql

```diff
@@ -0,0 +1,82 @@
+--
+-- Copyright (c) Microsoft Corporation.
+-- Licensed under the MIT License.
+--
+-- SQLite3/DuckDB version of the DB schema.
+
+DROP TABLE IF EXISTS trial_telemetry;
+DROP TABLE IF EXISTS trial_results;
+DROP TABLE IF EXISTS trial_config;
+DROP TABLE IF EXISTS experiment_merge;
+DROP TABLE IF EXISTS trial_status;
+DROP TABLE IF EXISTS experiment_config;
+
+CREATE TABLE experiment_config (
+    exp_id VARCHAR(128) NOT NULL,
+    descr TEXT,
+    git_repo VARCHAR(255) NOT NULL,
+    git_commit VARCHAR(40) NOT NULL,
+
+    PRIMARY KEY (exp_id)
+);
+
+CREATE TABLE trial_status (
+    exp_id VARCHAR(128) NOT NULL,
```

Comment:
```suggestion
    exp_id experiment_id_t NOT NULL,
```
Not using the type defined above.
Or at least bump it up to 255 for all of these I think?

Reply:
Nm, that's a different file, so only the latter comment applies I guess.